python/heterocl/tvm/build_module.py

```diff
@@ -107,6 +107,7 @@ def tvm_callback_exec_evaluate(platform, mode):
       cmd = "cd project; " + \
             "XCL_EMULATION_MODE=sw_emu ./host build_dir" + \
             ".sw_emu." + device + "/kernel.xclbin"
+      if empty: cmd = "cd __tmp__; ./host"
```

Comment:
We are not using `__tmp__` anymore, right?

Reply:
We absolutely need to avoid hardcoding paths or directory names in our code. We should have an API to query such information instead.

Reply:
It's fixed. We are not using any hardcoding abs path right now.